libs/pika/execution_base/tests/unit/any_sender.cpp

```diff
@@ -11,8 +11,11 @@
 #include <pika/modules/errors.hpp>
 #include <pika/testing.hpp>
 
+#include "algorithm_test_utils.hpp"
```

Comment:
Could you please move it to a subdirectory?
```suggestion
#include <pika/execution_base/tests/algorithm_test_utils.hpp>
```
Don't forget to update the tests in `execution` to use this header as well.